src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Each customer has his own card that serves as an additional identifier.
+ */
+public final class LoyaltyCard {
+    /**
+     * Customer's card id.
+     */
+    private final String number;
+
+    /**
+     * Constructor creates a new card with the passed number.
+     *
+     * <p>Checking the card number.
+     *
+     * @param number a unique card number, consisting of digital characters.
+     */
+    public LoyaltyCard(CharSequence number) {
+        this.number = checkNotNull(number);
+        this.number = number;
+    }
+
+    /**
+     * Method to get the card id if the customer provides a card or its number.
```

Comment:
Of course, it's a method. I see it from the code below. Say it shorter, and to the point.